api/src/pcapi/routes/pro/venues.py

```diff
@@ -79,8 +79,14 @@ def get_venues(query: venues_serialize.VenueListQueryModel) -> venues_serialize.
         offerer_id=offerer_id,
         validated_offerer=query.validated,
     )
+    ids_of_venues_with_offers = (
+        offerers_repository.get_venues_with_offers(venue_list[0].managingOffererId) if len(venue_list) > 0 else []
```

Comment:
(style) J'insiste (un tout petit peu), on raccourcit souvent la condition "est-ce que ma liste est vide ou non" comme suit : 

```suggestion
        offerers_repository.get_venues_with_offers(venue_list[0].managingOffererId) if venue_list else []
```

Reply:
@dbaty Avec les suggestions, je me retrouve avec l'erreur `error: Unsupported operand types for > ("List[Venue]" and "int")`

Reply:
Avec la proposition du dessus, il restait le `> 0` ce qui causait une erreur. Corrigé.